helm/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name | default "pccs"  }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name | default "pccs" }}
+    app.kubernetes.io/version: {{ trimPrefix "v" (index (split ":" .Values.image) 1) }}
+    app.kubernetes.io/managed-by: Helm
```

Comment:
The labels must follow https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels docs.

Also, use tpl to define it and just import it in all labels and selectortLabels points...